index.html

```diff
@@ -80,7 +80,7 @@ <h3>At-risk features</h3>
         <h3>HTML core specifications</h3>
         <p>Devices MUST be conforming implementations of the following specifications:</p>
         <ul>
-          <li> HTML 5.1 [[!HTML51]], devices acting as Web browsers that support the HTML syntax and the XHTML syntax, scripting, and the suggested default rendering.</li>
+          <li> HTML 5.2 [[!HTML52]]</li>
```

Comment:
The update drops the rest of the text, which was meant to identify the [HTML conformance classes](https://www.w3.org/TR/html52/infrastructure.html#conformance-classes) that devices must implement, as discussed in #21, notably to mandate scripting support.

Perhaps this goes without saying. A user agent that supports EcmaScript, as mandated by the Web Media API spec, without supporting the `<script>` tag clearly does not make any sense.

Reply:
Now I remember. I took it out because I didn't recall that. I'm not sure it's necessary, but we can put it back.

I think it might be clearer as an indented bullet, like we do for exceptions. It would also be clearer if we explicitly mention "conformance class".

Reply:
@tidoust I added a bullet on conformance class. Is wording OK? Also, I used code style for conformance class name and I'm pretty sure that's the wrong formatting.

I'm open to suggestions on wording and formatting.